glue_jupyter/bqplot/image/frb_mark.py

```diff
@@ -56,6 +56,12 @@ def update(self, *args, **kwargs):
         ymin = self.viewer.figure.axes[1].scale.min
         ymax = self.viewer.figure.axes[1].scale.max
 
+        # Expand beyond the boundary
```

Comment:
What does this mean when the whole image already fits in the view? Does it do unnecessary computation in that case, or is this smart enough?

Reply:
It is smart enough :)